### PR TITLE
libqt5pas, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-qt/libqt5pas/libqt5pas-1.2.9.recipe [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	devel:libQt5Pas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libqt5pas$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libqt5Core$secondaryArchSuffix
+	lib:libqt5Gui$secondaryArchSuffix
```
Comment (korli) [inline dev-qt/libqt5pas/libqt5pas-1.2.9.recipe:41]: Are you sure with lib:?

Comment (Begasus) [reply dev-qt/libqt5pas/libqt5pas-1.2.9.recipe:41]: jikes :) strangly enough it didn't error out there ;)

Comment (Begasus) [reply dev-qt/libqt5pas/libqt5pas-1.2.9.recipe]: Thanks ! Fixed now :)

File: dev-qt/libqt5pas/libqt5pas-1.2.9.recipe [hunk 2]
```diff
@@ -0,0 +1,66 @@
+SUMMARY="Pascal wrapper for Qt5"
+DESCRIPTION="libqt5pas is an interface between Lazarus and the Qt5 libraries.
+Not all Qt5 functions are available, only those necessary for Lazarus \
+functionality."
+HOMEPAGE="https://wiki.freepascal.org/Qt5_Interface"
+COPYRIGHT="2005-2013 by Jan Van hijfte"
```
Comment (Begasus) [inline dev-qt/libqt5pas/libqt5pas-1.2.9.recipe:6]: Not sure here, I see Jan as the original maintainer for libqt4pas, but I think Željan should be added as the current one (can't find it atm)

Comment (Begasus) [reply dev-qt/libqt5pas/libqt5pas-1.2.9.recipe:6]: Found it :)

File: dev-qt/libqt5pas/libqt5pas-1.2.9.recipe [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+INSTALL()
+{
+	#hack to fix the install path
+	if [ "effectiveTargetArchitecture" = x86_64 ]; then
+		sed -i "s|/boot/system/lib||" Makefile
+	else
+		sed -i "s|/boot/system/lib/x86||" Makefile
+	fi
```
Comment (Begasus) [inline dev-qt/libqt5pas/libqt5pas-1.2.9.recipe:66]: `target.path = $$[QT_INSTALL_LIBS]` seems to hardcode the patch to the Makefile, haven't found another way to fix this for now

**Comment by Begasus** [comment]
Not the best solution atm, but it's working for now ...

**Pull request merged**